tests/foreman/ui/test_user.py

```diff
@@ -1329,3 +1330,436 @@ def test_positive_create_in_ldap_mode(self):
                 password2='',
             )
             self.assertIsNotNone(self.user.search(user_name))
+
+
+class SshKeyInUser(UITestCase):
+    """Implements the SSH Key in User Tests"""
+
+    @stubbed
+    @tier2
+    def test_postitive_ssh_key_tab_presence(self):
+        """SSH keys tab presence in User Template
```

Comment:
do we really need to cover tab presence as a test ? I think all operations would only be performed by selecting the tab so I don't think we need a separate test. what do think @jyejare ?